packages/core/src/middleware/koa-pagination.ts

```diff
@@ -0,0 +1,63 @@
+import { MiddlewareType } from 'koa';
+import { number } from 'zod';
+
+import RequestError from '@/errors/RequestError';
+import { buildLink } from '@/utils/pagination';
+
+export interface Pagination {
+  page: number; // Begin from 1
+  size: number;
```

Comment:
use `perPage` to keep aligning with link?

Reply:
use `limit` instead